bitcoin/src/address/mod.rs

```diff
@@ -193,7 +193,7 @@ impl fmt::Display for AddressInner {
 pub enum KnownHrp {
     /// The main Bitcoin network.
     Mainnet,
-    /// The test networks, testnet and signet.
+    /// The test networks, testnet(testnet3) testnet4 and signet.
```

Comment:
In ae178a392f9774f31195aff3b49830e9c00fb3e7:

nit: should have space before `(testnet3)` and a comma after.